dashboard/app/jobs/aichat_request_chat_completion_job.rb

```diff
@@ -44,28 +47,38 @@ def perform(request:, locale:)
     request.update!(response: response, execution_status: status)
   end
 
+  private def get_toxicity_threshold_user_input
+    DCDO.get("aws_comprehend_toxicity_threshold_user_input", DEFAULT_TOXICITY_THRESHOLD_USER_INPUT)
```

Comment:
nit: can we make these keys aichat specific (i.e. `aichat_toxicity_threshold...`)? I'm thinking that if/when we start using Comprehend for other products, we may want to set separate global and product-specific thresholds, so it would be good to scope this to just aichat for now